.gitignore

```diff
@@ -1,2 +1,3 @@
 *.tar
 hadolint.json
+Makefile
```

Comment:
```suggestion
Makefile

```

Reply:
I'm being a bit pedantic here, but a newline should terminate each file as stated by POSIX in https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206 (a line is always ended by a newline character). There is a clear explanation here: https://thoughtbot.com/blog/no-newline-at-end-of-file . For visual studio, I've this page in my bookmarks to help configuring my editor on new machines, hope ot helps :)